azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -68,6 +85,9 @@ def _default_options(cls) -> Options:
             },
             targetCapability="BasicExecution",
             )
+    
+    def gateset(self):
```

Comment:
For all QIR backends the `gateset` configuration property should be `qis`. I think this can be done the same way that `basis_gates` is done, which is by creating a private `_gateset` method in the `AzureQirBackend` class and using it when populating the configuration.

Reply:
the notion of `gateset` as a parameter only really exists in IonQ's backends... because of this I added the gateset method in IonQ's QIR backend base. in fact, the gate set field is not actually even used for anything in the QIR IonQ backend... I only added it to have consistency in flags across IonQ backends, and to keep a command like `get_backend("ionq.simulator", gateset="qis')` non-breaking while still pointing to a QIR backend. For now, I will leave it as is. Let me know if we prefer to move it, or remove it, or leave.